index.js

```diff
@@ -63,6 +65,26 @@ function fastifyNext (fastify, options, next) {
         })
     }
   }
+
+  async function render (request, reply) {
+    reply.nextRender = async path => {
```

Comment:
I meant this closure should be a top level one. You can remove the onRequest hook.

Reply:
I need access to the request and reply, though. How would I get those without the hook?

Reply:
`reply.nextRender()` will call `nextRender()` with `this` as the `reply` (if you drop the arrow function). Then you have `reply.request` available as well.

Reply:
Ah! I didn't know that. So I can just do `fastify.decorateReply('nextRender', nextRender)`? That's really nice!

Reply:
[`48e7b03` (#201)](https://github.com/fastify/fastify-nextjs/pull/201/commits/48e7b037b0ab5c94225afabab3ef26bd2bbcf8b1)

Reply:
> Ah! I didn't know that. So I can just do `fastify.decorateReply('nextRender', nextRender)`? That's really nice!

yes you can!